_posts/2024-02-22-replacing-default-admin-credentials.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title:  "Enhancing Security in OpenSearch 2.12.0: The End of Default Admin Password"
+authors:
+ - dchanp
+date: 2024-02-22
+categories:
+ - community-updates
+ - technical-posts
+meta_keywords: "OpenSearch Security, Security Demo Configuration"
+meta_description: Learn about the changes to Security plugin's demo configuration setup
+has_math: false
+has_science_table: false
+---
+
+
+In the landscape of data management and search technologies, OpenSearch stands out by offering a comprehensive set of features and a flexible platform for users globally. With the rollout of OpenSearch version 2.12.0, an update aimed at improving the initial setup process for "admin" user using the Security plugin's demo configuration tool has been introduced.
```

Comment:
Considering link from the mention of 2.12.0 to the release notes here: https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.12.0.md